src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const obj = state;
+
+  actions.forEach(element => {
```

Comment:
destructurize element

Reply:
Task:
`state` is an object. You are supposed to add, change, or delete its
  properties instead of creating a new object.
  
 maybe I don't understand something? 
 `destructurize element` what for?